.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
+button, please make sure you check off all the following boxes:
+
+- [ ] All existing unit tests pass in all supported Python versions (2.6, 2.7,
+      and 3.3+)
+- [ ] New tests have been added to verify the new functionality / bug fix
```

Comment:
I like this point. Let's also add an extra checkbox having them explain why they are adding this and their use case. I occasionally find myself pinging a PR to learn more about what their use case may be.